backend/src/main/java/com/example/mazerunner/Heuristic.java

```diff
@@ -0,0 +1,27 @@
+package com.example.mazerunner;
+
+public interface Heuristic {
+    /**
```

Comment:
```suggestion
    private double[][] heuristic
```
check if it make sense to add heuristic array into Heuristic interface